### DISCUSSION TITLE
Trying to connect AOSP TV

### DISCUSSION THREAD

**Discussion by DeliDumrul43** [yume-chan/ya-webadb#471] [Q&A] [answered]
Hi , 

I am struggling to understand that how can I connect to my device ? I follow this https://github.com/yume-chan/ya-webadb/discussions/245#discussioncomment-384030 issue comments but I guess this is for emulator devices .

Any help ?

**Comment by yume-chan** [comment, marked as answer]
That method can connect to any device with ADB over WiFi enabled. Have you enabled that and what exact steps did you do?

**Reply by DeliDumrul43** [reply]
I was misunderstood the working order of this software now I can able to connect my AOSP TV 
Thanks

**Comment by upintheairsheep** [comment]
No
